packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -271,7 +282,26 @@ class ListAddMoreQuestion extends Question {
 		}
 
 		// save
-		const responseToSave = await this.getDataToSave(req, journeyResponse);
+		let responseToSave = await this.getDataToSave(req, journeyResponse);
+		journey.response[this.fieldName] = responseToSave.answers[this.fieldName];
+
+		if (this.subQuestion instanceof AddressAddMoreQuestion) {
+			const addresses = responseToSave.answers[this.fieldName];
+			await Promise.all(
```

Comment:
could this be in its own function
part of me wants to pass it into the question constructor but maybe that's too much for now

Reply:
Not to be addressed in this PR